app/models/user.rb

```diff
@@ -101,9 +101,12 @@ def with_team_kind(kind)
     end
 
     def with_all_associations_joined
-      includes(:conferences).group("conferences.id").
-      includes(:roles).group("roles.id").
-      includes(roles: :team).group("teams.id")
+      eager_load(:conferences).
```

Comment:
The `group` methods are unrelated to eager loading, they simply group the records by the given columns. These clauses were most likely added to ensure only unique records were returned.